GOVERNANCE.md

```diff
@@ -0,0 +1,50 @@
+OpenWDL Governance and Process
+==============================
+
+Table of Contents
+-----------------
+[Table of Contents]: #table-of-contents
+  - [Governance](#governance)
+  - [RFC Process](#rfc-process)
+  
+Governance
+----------
+
+WDL has a community-driven development process where most decisions are made through consensus, under the stewardship of a 'core team'. The core team is responsible for steering the design and development process, overseeing the introduction of new features, setting priorities and release schedule, and ultimately making decisions for the rare instances where there is no community consensus . 
```

Comment:
I know you've already merged, but FYI there's an extra space before the last . at the end of the paragraph.
> there is no community **consensus .**